WRITING.md

```diff
@@ -1,238 +1,127 @@
+We want to present a consistent and professional face to our users and partners.
 
-**Contents**
+Of course, trends in documentation styling change over time, so we're fully aware that these guidelines will likely shift as we consider adopting new best practices.
 
-- [When do I need to follow this?](#when-do-i-need-to-follow-this)
-- [Writing to teach](#writing-to-teach)
-- [Content types](#content-types)
-- [Tone and style](#tone-and-style)
-- [Accessibility](#accessibility)
-- [Terminology](#terminology)
+### Layout
 
-## When do I need to follow this?
+### Code usage examples
 
-You can make a content PR _without_ following this guide. First draft PRs do not have to be mindful of tone, accessibility, or terminology. You do not need to be a great English speaker to contribute to the English docs. (Try to follow the methodology in [content types](#content-types) section so your PR matches our expectations for its content type!)
+### Images
 
-Feel free to grab an Issue and make a PR using rough paragraphs or bullet points. We'll review your PR and make the necessary edits so that the language is friendly, accessible, and accurate. That editing process must follow this guide.
+## Organization
 
-On the flip side, not all of our existing content (and certainly not all new PRs), will follow these guidelines effectively. Another helpful way to contribute is by editing content to follow this writing guide.
+### Headings
 
-If you'd like to contribute to this guide itself, feel free to submit an Issue or PR with the [governance](https://github.com/solidjs/solid-docs-next/labels/governance) tag.
+### Lists
 
-## Writing to teach
+### Tool tips
 
-The goal of documentation has often been to _describe_ the technology. If you can describe how everything works, someone can read what you wrote and gain the understanding they need to use the tech. This usually takes a lot of effort on the reader's part: we're all familiar with docs that are just _too_ technical for us to easily understand.
+### Page linking
 
-Instead, we wish to deliberately _teach_: create resources that are designed to most effectively promote learning. To do this, we incorporate - as best we can - an understanding of our users and how people learn.
+## Voice and tone
 
-<details>
-  <summary>
-  1. Have someone in mind when you write
-  </summary>
+So far, we've dealt with how your contributions should look and function.
+As important as all of that is, the core part of your documents is the writing itself, so we want to discuss how we would like our documentation to read.
 
-We're writing for people, and different people come to the docs at different levels of experience and expectations. This doesn't mean that every piece of writing should serve every learning need - see our different [Content Types](#content-types). But it does mean that you can teach more effectively if you have an idea of whom you're teaching and what works for them.
+### Approachable
```

Comment:
[Nitpick] I think it would be more beneficial to name this section Language and then you can give tips on how users can write in an approachable way.
```suggestion
### Language 
```